AssetEditor/AssetEditor.csproj

```diff
@@ -53,13 +55,13 @@
         <DebugType>embedded</DebugType>
         <Authors>AssetEdCommunity</Authors>
         <Company>Private</Company>
-        <Product>AssetEditor</Product>
+        <Product>AssetEditor45-Phazt{Tester of FBX Model impor/export}</Product>
         <PackageProjectUrl>https://github.com/donkeyProgramming/TheAssetEditor</PackageProjectUrl>
         <RepositoryUrl>https://github.com/donkeyProgramming/TheAssetEditor</RepositoryUrl>
         <RepositoryType>GIT</RepositoryType>
-        <AssemblyVersion>0.45.0.0</AssemblyVersion>
-        <FileVersion>0.45.0.0</FileVersion>
-        <PackageId>AssetEditor v0.45</PackageId>
+        <AssemblyVersion>0.2.0.0</AssemblyVersion>
+        <FileVersion>0.2.0.0</FileVersion>
+        <PackageId>AssetEditor45-Phazt</PackageId>
```

Comment:
It's better idea to call it AssetEditor v.05 instead to avoid any confusion with old userbase. Also for historical reason.

Reply:
Related issue (low prio) https://github.com/admiralnelson/TheAssetEditor/issues/5

Reply:
Update from discord convo: it should be called AssetEditor45 (45 because we started on version 45) AssetEditor (because we want to keep it generic and open)
[Resolve this and then I can merge it]